src/PokemonGymOwner.java

```diff
@@ -0,0 +1,33 @@
+import java.util.List;
+
+public class PokemonGymOwner {
+    private String name = "";
+    private String town = "";
+    private List<Pokemon> pokemons;
```

Comment:
Waarom instantieer de Strings wel met een lege String (name = ""), maar de initieer je de List niet met een lege ArrayList? Als je dat niet doet, kun je tegen NullPointerExceptions aanlopen. Bijvoorbeeld als je de .add() methode aanroept terwijl de List eigenlijk null is.